src/gui/src/layoutViewer.h

```diff
@@ -261,6 +266,8 @@ class LayoutViewer : public QWidget, public odb::dbBlockCallBackObj
   QPoint centering_shift_;
 
   static constexpr qreal zoom_scale_factor_ = 1.2;
+
+  const QColor background_ = Qt::black;
```

Comment:
The default was black so I'm curious why you made it explicit.

Reply:
It was needed in two places, so I thought it was easier to make it a common variable (one in paintEvent and one in saveImage)